frontend/App.tsx

```diff
@@ -1,17 +1,13 @@
 import { StatusBar } from "expo-status-bar";
 import React from "react";
 import { StyleSheet, Text, View } from "react-native";
-import Amplify from "aws-amplify";
-import awsconfig from "./aws-exports";
-Amplify.configure(awsconfig);
+import LoginScreen from './src/pages/LoginPage';
+// import Amplify from "aws-amplify";
```

Comment:
Was this causing it to crash - sorry my bad I shouldn't have merged this in without checking!